api/README.md

```diff
@@ -0,0 +1,3 @@
+# API Service
```

Comment:
Can you change the title to be the same as the title of the linked docs page (like "Django API" in this case)? I mean this for all 4 files.